services/src/main/java/org/keycloak/forms/account/freemarker/model/RealmBean.java

```diff
@@ -72,4 +72,8 @@ public boolean isRegistrationEmailAsUsername() {
     public boolean isUserManagedAccessAllowed() {
         return realm.isUserManagedAccessAllowed();
     }
+
+    public boolean isUserDeleteOwnAccountAllowed() {
```

Comment:
This option is not needed at all. By default users shouldn't have the delete-account role associated with them and if they have it then they can delete their account, so there is no need for an additional config option in the realm.